### ISSUE TITLE
Simulations sidebar: accordion controls lack keyboard focus indicators

### ISSUE THREAD

**Issue by eatyourgreens** [pkpdapp-team/pkpdapp#589]
Here's a quick screen recording in Chrome, where I use the Tab key to quickly navigate through the sidebars. All the interactive controls have the default Material UI focus indicators (👍) except for the collapsed accordion controls in the Simulations sidebar. Each of them should have some visible indicator when it has keyboard focus, so that you can see where you are when you navigate the page with Tab/shift-Tab.

[WCAG 2.2, Success Criterion 2.4.7: Focus Visible (level AA)](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html).

https://github.com/user-attachments/assets/fbdaa311-d571-4895-94fc-767ee8f3d0ca
